src/IndustrialRobot/IndustrialRobot/ViewModels/RelayCommand.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.Ports;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+
+namespace IndustrialRobot.ViewModels
+{
+    public class RelayCommand : ICommand
```

Comment:
zakladam ze to copy paste :) tez bym skopiowal, ale daj znaka jak dobrze rozumiesz ten kod??

Reply:
Copy paste nie całkiem zadziałał :) Autor zostawił sporo pułapek, więc trzeba było zachować czujność.
Wracając do pytania, zależy jak dobrze powinienem go rozumieć. Samo działanie jako takie prześledziłem krok po kroku, ale nie ukrywam, że wiele zagadnień jest tu dla mnie nowych (Action<object>, Func<>, (object parameter) i CommandManager.RequerySuggested).